### PR TITLE
translation: monitor、tag-transmission

### PR DESCRIPTION
**Pull request by daizhenyu**
**What type of PR is this?**

Doc.

**What this PR does / why we need it?**

This initiative aims to enhance readability, accessibility, and collaboration for both current and potential contributors, ensuring a unified language throughout the codebase.

**Which issue(s) this PR fixes？**

Fixes #1466 

**Does this PR introduce a user-facing change?**

No

## Checklist
- [x] Make sure there is a [GitHub_issue](https://github.com/huaweicloud/Sermant/issues) related with this PR before you start working on it.
- [x] Make sure you have squashed your change to one single commit.
- [x] GitHub Actions works fine in this PR.

### CODE REVIEW

File: sermant-plugins/sermant-monitor/monitor-plugin/src/main/java/com/huawei/monitor/declarer/AbstractDeclarer.java [hunk 1]
```diff
@@ -35,11 +35,11 @@ public abstract class AbstractDeclarer extends AbstractPluginDeclarer {
     private final String methodName;
 
     /**
-     * 构造方法
+     * constructionMethod
```
Comment (lilai23) [inline sermant-plugins/sermant-monitor/monitor-plugin/src/main/java/com/huawei/monitor/declarer/AbstractDeclarer.java]: constructor

File: sermant-plugins/sermant-monitor/monitor-plugin/src/main/java/com/huawei/monitor/declarer/AbstractDeclarer.java [hunk 2]
```diff
@@ -35,11 +35,11 @@ public abstract class AbstractDeclarer extends AbstractPluginDeclarer {
...
      *
-     * @param enhanceClass 增强类
-     * @param interceptClass 拦截类
-     * @param methodName 增强方法
+     * @param enhanceClass enhancementClass
+     * @param interceptClass interceptClass
+     * @param methodName enhancementMethod
```
Comment (lilai23) [inline sermant-plugins/sermant-monitor/monitor-plugin/src/main/java/com/huawei/monitor/declarer/AbstractDeclarer.java]: insert blank

File: sermant-plugins/sermant-monitor/monitor-service/src/main/java/com/huawei/monitor/command/CommonMonitorCommand.java [hunk 3]
```diff
@@ -29,12 +29,12 @@
 import java.util.logging.Logger;
 
 /**
- * 结果解析抽象类
+ * results parse abstract classes
```
Comment (lilai23) [inline sermant-plugins/sermant-monitor/monitor-service/src/main/java/com/huawei/monitor/command/CommonMonitorCommand.java]: result parsing abstract class

File: sermant-plugins/sermant-monitor/monitor-service/src/main/java/com/huawei/monitor/command/CommonMonitorCommand.java [hunk 4]
```diff
@@ -61,9 +61,9 @@ protected List<String> readLines(InputStream inputStream) {
     }
 
     /**
-     * 错误前置处理
+     * errorPreprocessing
```
Comment (lilai23) [inline sermant-plugins/sermant-monitor/monitor-service/src/main/java/com/huawei/monitor/command/CommonMonitorCommand.java]: add blank

File: sermant-plugins/sermant-monitor/monitor-service/src/main/java/com/huawei/monitor/command/MonitorCommand.java [hunk 5]
```diff
@@ -19,34 +19,34 @@
...
 public interface MonitorCommand<T> {
 
     /**
-     * 返回Linux指令
+     * return linux command
      *
-     * @return linux指令
+     * @return linux command
      */
     String getCommand();
```
Comment (lilai23) [inline sermant-plugins/sermant-monitor/monitor-service/src/main/java/com/huawei/monitor/command/MonitorCommand.java]: get linux command

File: sermant-plugins/sermant-monitor/monitor-service/src/main/java/com/huawei/monitor/command/NetworkCommand.java [hunk 6]
```diff
@@ -77,39 +77,39 @@ public NetDev parseResult(InputStream inputStream) {
...
 
         /**
-         * 写包速度
+         * write packet speed
          */
         private final long transmitPackets;
 
         /**
-         * 构造方法
+         * constructionMethod
```
Comment (lilai23) [inline sermant-plugins/sermant-monitor/monitor-service/src/main/java/com/huawei/monitor/command/NetworkCommand.java]: constructor

File: sermant-plugins/sermant-monitor/monitor-service/src/main/java/com/huawei/monitor/common/MetricEnum.java [hunk 7]
```diff
@@ -17,427 +17,427 @@
...
 
     /**
-     * codeCache提交值
+     * value submitted by codeCache
      */
     CODE_CACHE_COMMITTED("code_cache_committed", "the committed is init of code cache"),
 
     /**
-     * meta space 初始值
+     * meta space initialValue
```
Comment (lilai23) [inline sermant-plugins/sermant-monitor/monitor-service/src/main/java/com/huawei/monitor/common/MetricEnum.java]: initial value

Comment (lilai23) [inline sermant-plugins/sermant-monitor/monitor-service/src/main/java/com/huawei/monitor/common/MetricEnum.java]: description

File: sermant-plugins/sermant-tag-transmission/config/config.yaml [hunk 8]
```diff
@@ -1,21 +1,27 @@
...
-  # 是否需要在new Thread时透传流量标签，默认开启
+  # Whether to pass traffic labels when a new Thread is created. This parameter is enabled by default
   enabled-thread: true
-  # 是否需要在普通线程池创建新的线程任务时透传流量标签，默认开启
+  # Whether to pass traffic labels when creating new thread tasks in a common thread pool.
+  # This parameter is enabled by default
   enabled-thread-pool: true
-  # 是否需要在定时线程池创建新的线程任务时透传流量标签，默认开启
+  # Whether to pass traffic labels when new thread tasks are created in the scheduled
+  # thread pool. This parameter is enabled by default
```
Comment (lilai23) [inline sermant-plugins/sermant-tag-transmission/config/config.yaml]: transmit not pass, check all in this file

File: sermant-plugins/sermant-tag-transmission/tag-transmission-httpclient4.x-plugin/src/main/java/com/huaweicloud/sermant/tag/transmission/httpclientv4/interceptors/HttpClient4xInterceptor.java [hunk 9]
```diff
@@ -55,9 +55,9 @@ public ExecuteContext doAfter(ExecuteContext context) {
     }
 
     /**
-     * 向HttpRequest中添加流量标签
+     * add traffic tag to the http method
```
Comment (lilai23) [inline sermant-plugins/sermant-tag-transmission/tag-transmission-httpclient4.x-plugin/src/main/java/com/huaweicloud/sermant/tag/transmission/httpclientv4/interceptors/HttpClient4xInterceptor.java]: add traffic tag to HttpRequest

File: sermant-plugins/sermant-tag-transmission/tag-transmission-servicecomb-plugin/src/test/java/com/huaweicloud/sermant/tag/transmission/servicecomb/interceptors/ServiceCombRpcConsumerInterceptorTest.java [hunk 10]
```diff
@@ -46,38 +46,38 @@ public void doBefore(TrafficTag trafficTag) {
...
-        // TrafficTag只有部分tag信息
+        // traffic tag indicates only partial tag information
         context = buildContext(new Invocation());
         TrafficUtils.getTrafficTag().getTag().remove("id");
         returnContext = interceptor.before(context);
         expectContext = buildExpectContext("name");
         Assert.assertEquals(((Invocation) returnContext.getArguments()[0]).getContext(), expectContext);
 
-        // TrafficTa没有tag信息
+        // traffic tag there is no tag information
```
Comment (lilai23) [inline sermant-plugins/sermant-tag-transmission/tag-transmission-servicecomb-plugin/src/test/java/com/huaweicloud/sermant/tag/transmission/servicecomb/interceptors/ServiceCombRpcConsumerInterceptorTest.java]: there is no tag information in TrafficTag

File: sermant-plugins/sermant-tag-transmission/tag-transmission-servicecomb-plugin/src/test/java/com/huaweicloud/sermant/tag/transmission/servicecomb/interceptors/ServiceCombRpcConsumerInterceptorTest.java [hunk 11]
```diff
@@ -46,38 +46,38 @@ public void doBefore(TrafficTag trafficTag) {
...
+        // The traffic label pass-through switch is turned off
         tagTransmissionConfig.setEnabled(false);
         context = buildContext(new Invocation());
         returnContext = interceptor.before(context);
         expectContext = buildExpectContext();
         Assert.assertEquals(((Invocation) returnContext.getArguments()[0]).getContext(), expectContext);
         tagTransmissionConfig.setEnabled(true);
 
-        // TrafficTag包含完整的tag信息
+        // the traffic tag contains complete tag information
```
Comment (hanbingleixue) [inline sermant-plugins/sermant-tag-transmission/tag-transmission-servicecomb-plugin/src/test/java/com/huaweicloud/sermant/tag/transmission/servicecomb/interceptors/ServiceCombRpcConsumerInterceptorTest.java]: TrafficTag

File: sermant-plugins/sermant-tag-transmission/tag-transmission-rocketmq5.x-plugin/src/test/java/com/huaweicloud/sermant/tag/transmission/rocketmqv5/interceptor/RocketmqConsumerInterceptorTest.java [hunk 12]
```diff
@@ -113,7 +113,7 @@ public void testRocketmqConsumer() {
         Assert.assertNull(TrafficUtils.getTrafficTag().getTag().get("id"));
         Assert.assertEquals("testName001", TrafficUtils.getTrafficTag().getTag().get("name").get(0));
 
-        // 测试TagTransmissionConfig开关关闭时
+        // Test when the Tag Transmission Config switch is off
```
Comment (hanbingleixue) [inline sermant-plugins/sermant-tag-transmission/tag-transmission-rocketmq5.x-plugin/src/test/java/com/huaweicloud/sermant/tag/transmission/rocketmqv5/interceptor/RocketmqConsumerInterceptorTest.java]: TagTransmissionConfig

File: sermant-plugins/sermant-tag-transmission/tag-transmission-rocketmq5.x-plugin/src/test/java/com/huaweicloud/sermant/tag/transmission/rocketmqv5/interceptor/RocketmqProducerInterceptorTest.java [hunk 13]
```diff
@@ -52,7 +52,7 @@ public void test() {
         Map<String, List<String>> tags = new HashMap<>();
         TrafficUtils.removeTrafficTag();
 
-        // 测试TagTransmissionConfig开关关闭时
+        // Test when the Tag Transmission Config switch is off
```
Comment (hanbingleixue) [inline sermant-plugins/sermant-tag-transmission/tag-transmission-rocketmq5.x-plugin/src/test/java/com/huaweicloud/sermant/tag/transmission/rocketmqv5/interceptor/RocketmqProducerInterceptorTest.java]: TagTransmissionConfig

File: sermant-plugins/sermant-tag-transmission/tag-transmission-httpserver-plugin/src/main/java/com/huaweicloud/sermant/tag/transmission/server/interceptors/HttpServletInterceptor.java [hunk 14]
```diff
@@ -86,16 +86,17 @@ public ExecuteContext onThrow(ExecuteContext context) {
     }
 
     /**
-     * 从HttpServletRequest中解析流量标签
+     * Parse the traffic tag from the Http Servlet Request
```
Comment (hanbingleixue) [inline sermant-plugins/sermant-tag-transmission/tag-transmission-httpserver-plugin/src/main/java/com/huaweicloud/sermant/tag/transmission/server/interceptors/HttpServletInterceptor.java]: HttpServletRequest

File: sermant-plugins/sermant-tag-transmission/tag-transmission-httpclient3.x-plugin/src/main/java/com/huaweicloud/sermant/tag/transmission/httpclientv3/interceptors/HttpClient3xInterceptor.java [hunk 15]
```diff
@@ -55,9 +55,9 @@ public ExecuteContext doAfter(ExecuteContext context) {
     }
 
     /**
-     * 向HttpMethod中添加流量标签
+     * add traffic tag to the http method
```
Comment (hanbingleixue) [inline sermant-plugins/sermant-tag-transmission/tag-transmission-httpclient3.x-plugin/src/main/java/com/huaweicloud/sermant/tag/transmission/httpclientv3/interceptors/HttpClient3xInterceptor.java]: HttpMethod

File: sermant-plugins/sermant-monitor/monitor-plugin/src/main/java/com/huawei/monitor/declarer/AbstractDeclarer.java (hunk 1 above)
Comment (daizhenyu) [reply sermant-plugins/sermant-monitor/monitor-plugin/src/main/java/com/huawei/monitor/declarer/AbstractDeclarer.java]: done

File: sermant-plugins/sermant-monitor/monitor-plugin/src/main/java/com/huawei/monitor/declarer/AbstractDeclarer.java (hunk 2 above)
Comment (daizhenyu) [reply sermant-plugins/sermant-monitor/monitor-plugin/src/main/java/com/huawei/monitor/declarer/AbstractDeclarer.java]: done

File: sermant-plugins/sermant-tag-transmission/tag-transmission-dubbo2.7.x-plugin/src/main/java/com/huaweicloud/sermant/tag/transmission/apachedubbov2/interceptors/ApacheDubboConsumerInterceptor.java [hunk 16]
```diff
@@ -40,7 +40,7 @@ public class ApacheDubboConsumerInterceptor extends AbstractClientInterceptor<Rp
     private static final Logger LOGGER = LoggerFactory.getLogger();
 
     /**
-     * rpcInvocation参数在invoke方法的参数下标
+     * rpcInvocation parameter subscript
```
Comment (hanbingleixue) [inline sermant-plugins/sermant-tag-transmission/tag-transmission-dubbo2.7.x-plugin/src/main/java/com/huaweicloud/sermant/tag/transmission/apachedubbov2/interceptors/ApacheDubboConsumerInterceptor.java]: rpcInvocation parameter subscript of the invoke method

File: sermant-plugins/sermant-monitor/monitor-service/src/main/java/com/huawei/monitor/command/CommonMonitorCommand.java (hunk 3 above)
Comment (daizhenyu) [reply sermant-plugins/sermant-monitor/monitor-service/src/main/java/com/huawei/monitor/command/CommonMonitorCommand.java]: done

File: sermant-plugins/sermant-monitor/monitor-service/src/main/java/com/huawei/monitor/command/CommonMonitorCommand.java (hunk 4 above)
Comment (daizhenyu) [reply sermant-plugins/sermant-monitor/monitor-service/src/main/java/com/huawei/monitor/command/CommonMonitorCommand.java]: done

File: sermant-plugins/sermant-monitor/monitor-service/src/main/java/com/huawei/monitor/command/MonitorCommand.java (hunk 5 above)
Comment (daizhenyu) [reply sermant-plugins/sermant-monitor/monitor-service/src/main/java/com/huawei/monitor/command/MonitorCommand.java]: done

File: sermant-plugins/sermant-monitor/monitor-service/src/main/java/com/huawei/monitor/command/NetworkCommand.java (hunk 6 above)
Comment (daizhenyu) [reply sermant-plugins/sermant-monitor/monitor-service/src/main/java/com/huawei/monitor/command/NetworkCommand.java]: done

File: sermant-plugins/sermant-monitor/monitor-service/src/main/java/com/huawei/monitor/common/MetricEnum.java (hunk 7 above)
Comment (daizhenyu) [reply sermant-plugins/sermant-monitor/monitor-service/src/main/java/com/huawei/monitor/common/MetricEnum.java]: done

File: sermant-plugins/sermant-tag-transmission/config/config.yaml (hunk 8 above)
Comment (daizhenyu) [reply sermant-plugins/sermant-tag-transmission/config/config.yaml]: done

File: sermant-plugins/sermant-monitor/monitor-service/src/main/java/com/huawei/monitor/common/MetricEnum.java (hunk 7 above)
Comment (daizhenyu) [reply sermant-plugins/sermant-monitor/monitor-service/src/main/java/com/huawei/monitor/common/MetricEnum.java]: done

File: sermant-plugins/sermant-tag-transmission/tag-transmission-httpclient4.x-plugin/src/main/java/com/huaweicloud/sermant/tag/transmission/httpclientv4/interceptors/HttpClient4xInterceptor.java (hunk 9 above)
Comment (daizhenyu) [reply sermant-plugins/sermant-tag-transmission/tag-transmission-httpclient4.x-plugin/src/main/java/com/huaweicloud/sermant/tag/transmission/httpclientv4/interceptors/HttpClient4xInterceptor.java]: done

File: sermant-plugins/sermant-tag-transmission/tag-transmission-servicecomb-plugin/src/test/java/com/huaweicloud/sermant/tag/transmission/servicecomb/interceptors/ServiceCombRpcConsumerInterceptorTest.java (hunk 10 above)
Comment (daizhenyu) [reply sermant-plugins/sermant-tag-transmission/tag-transmission-servicecomb-plugin/src/test/java/com/huaweicloud/sermant/tag/transmission/servicecomb/interceptors/ServiceCombRpcConsumerInterceptorTest.java]: done

File: sermant-plugins/sermant-tag-transmission/tag-transmission-servicecomb-plugin/src/test/java/com/huaweicloud/sermant/tag/transmission/servicecomb/interceptors/ServiceCombRpcConsumerInterceptorTest.java (hunk 11 above)
Comment (daizhenyu) [reply sermant-plugins/sermant-tag-transmission/tag-transmission-servicecomb-plugin/src/test/java/com/huaweicloud/sermant/tag/transmission/servicecomb/interceptors/ServiceCombRpcConsumerInterceptorTest.java]: done

File: sermant-plugins/sermant-tag-transmission/tag-transmission-rocketmq5.x-plugin/src/test/java/com/huaweicloud/sermant/tag/transmission/rocketmqv5/interceptor/RocketmqConsumerInterceptorTest.java (hunk 12 above)
Comment (daizhenyu) [reply sermant-plugins/sermant-tag-transmission/tag-transmission-rocketmq5.x-plugin/src/test/java/com/huaweicloud/sermant/tag/transmission/rocketmqv5/interceptor/RocketmqConsumerInterceptorTest.java]: done

File: sermant-plugins/sermant-tag-transmission/tag-transmission-rocketmq5.x-plugin/src/test/java/com/huaweicloud/sermant/tag/transmission/rocketmqv5/interceptor/RocketmqProducerInterceptorTest.java (hunk 13 above)
Comment (daizhenyu) [reply sermant-plugins/sermant-tag-transmission/tag-transmission-rocketmq5.x-plugin/src/test/java/com/huaweicloud/sermant/tag/transmission/rocketmqv5/interceptor/RocketmqProducerInterceptorTest.java]: done

File: sermant-plugins/sermant-tag-transmission/tag-transmission-httpserver-plugin/src/main/java/com/huaweicloud/sermant/tag/transmission/server/interceptors/HttpServletInterceptor.java (hunk 14 above)
Comment (daizhenyu) [reply sermant-plugins/sermant-tag-transmission/tag-transmission-httpserver-plugin/src/main/java/com/huaweicloud/sermant/tag/transmission/server/interceptors/HttpServletInterceptor.java]: done

File: sermant-plugins/sermant-tag-transmission/tag-transmission-httpclient3.x-plugin/src/main/java/com/huaweicloud/sermant/tag/transmission/httpclientv3/interceptors/HttpClient3xInterceptor.java (hunk 15 above)
Comment (daizhenyu) [reply sermant-plugins/sermant-tag-transmission/tag-transmission-httpclient3.x-plugin/src/main/java/com/huaweicloud/sermant/tag/transmission/httpclientv3/interceptors/HttpClient3xInterceptor.java]: done

File: sermant-plugins/sermant-tag-transmission/tag-transmission-dubbo2.7.x-plugin/src/main/java/com/huaweicloud/sermant/tag/transmission/apachedubbov2/interceptors/ApacheDubboConsumerInterceptor.java (hunk 16 above)
Comment (daizhenyu) [reply sermant-plugins/sermant-tag-transmission/tag-transmission-dubbo2.7.x-plugin/src/main/java/com/huaweicloud/sermant/tag/transmission/apachedubbov2/interceptors/ApacheDubboConsumerInterceptor.java]: done

**Review by lilai23 — APPROVED** [review]

**Review by hanbingleixue — APPROVED** [review]

**Pull request merged**